pkg/controller/master/supportbundle/manager.go

```diff
@@ -43,6 +43,7 @@ func (m *Manager) Create(sb *harvesterv1.SupportBundle, image string) error {
 	logrus.Debugf("creating deployment %s with image %s", deployName, image)
 
 	pullPolicy := m.getImagePullPolicy()
+	namespaces := []string{sb.Namespace, "longhorn-system", "default"}
```

Comment:
The default namespace is not special now. Why do we want to set it here?

Reply:
Removed.
But anyway it's the [default namespaces](https://github.com/rancher/support-bundle-kit/blob/189e740444790dc3a3c7f96a49b4ae9ca2db181c/pkg/manager/cluster.go#L86) hard-coded in SBK for now.

Reply:
Does SBKit collect resources from these namespaces? If that's the case we also need kube-system and cattle-system

Reply:
Two namespaces lists will be merged. By default, SBKit collects common namespaces in Rancher.